lightning/src/chain/channelmonitor.rs

```diff
@@ -2855,12 +2855,6 @@ impl<Signer: EcdsaChannelSigner> ChannelMonitorImpl<Signer> {
 				self.their_cur_per_commitment_points = Some((commitment_number, their_per_commitment_point, None));
 			}
 		}
-		let mut htlcs = Vec::with_capacity(htlc_outputs.len());
-		for htlc in htlc_outputs {
-			if htlc.0.transaction_output_index.is_some() {
-				htlcs.push(htlc.0);
-			}
-		}
```

Comment:
Previously used to populate a hash map dropped in 44e48679fbac4240ee9347bbebbe2d3d023bc2f8.